api/envoy/config/filter/network/redis_proxy/v2/redis_proxy.proto

```diff
@@ -90,6 +90,25 @@ message RedisProxy {
     // this limit, then redirection will fail and the original redirection error will be passed
     // downstream unchanged. This limit defaults to 100.
     google.protobuf.UInt32Value max_upstream_unknown_connections = 6;
+
+    // All ReadPolicy settings except MASTER may return stale data because replication is
+    // asynchronous and requires some delay. You need to ensure that your application can tolerate
+    // stale data.
+    enum ReadPolicy {
+      // Default mode. Read from the current master node.
+      MASTER = 0;
+      // Read from the master, but if it is unavailable, read from replica nodes.
```

Comment:
How is a replica selected? Random? Can you provide more details? Same below for all the options in which we might select from multiple nodes.